### PR TITLE
Fixing RPM icon issue

### LINKED ISSUE #370: GTK app icon not set (only used in some edge cases)
the eduVPN gtk icon is not properly set, which is used in some edge cases like task switching in window manager.

### PR DESCRIPTION
**Pull request by gijzelaerr**
This PR:
* fixed #370
* updates build files to latest versions.

### CODE REVIEW

File: eduvpn.spec [hunk 1]
```diff
@@ -107,13 +107,13 @@ desktop-file-install %{buildroot}/%{_datadir}/applications/eduvpn-client.desktop
 %doc README.md
 %{_bindir}/letsconnect-cli
 %{_bindir}/letsconnect-gui
-%{_datarootdir}/applications/letsconnect-client.desktop
+%{_datarootdir}/applications/org.letsconnect.client.desktop
```
Comment (alvra) [inline eduvpn.spec:110]: It's called `org.letsconnect-vpn.client` in `variants.py`. Probably a good idea to use the id name everywere.

**Comment by alvra** [comment]
If Fedora 33 and Ubuntu 18 are no longer supported, the documentation should also be updated to reflect that.

File: eduvpn/settings.py [hunk 2]
```diff
@@ -29,11 +29,11 @@
     "RWRtBSX1alxyGX+Xn3LuZnWUT0w//B6EmTJvgaAxBMYzlQeI+jdrO6KF",
     "RWQKqtqvd0R7rUDp0rWzbtYPA3towPWcLDCl7eY9pBMMI/ohCmrS0WiM"
 ]
-EDUVPN_ICON = prefix + "/share/icons/hicolor/128x128/apps/eduvpn-client.png"
+EDUVPN_ICON = prefix + "/share/icons/hicolor/128x128/apps/org.eduvpn.client.png"
```
Comment (alvra) [inline eduvpn/settings.py:32]: A few lines up it says `org.eduvpn.app.linux`. I think there should be a single setting for this that gets interpolated everywhere.

File: eduvpn.spec (hunk 1 above)
Comment (gijzelaerr) [reply eduvpn.spec:110]: i agree, can you apply that change?

File: eduvpn/settings.py (hunk 2 above)
Comment (gijzelaerr) [reply eduvpn/settings.py:32]: is CLIENT_ID the same as application ID?

File: eduvpn.spec (hunk 1 above)
Comment (alvra) [reply eduvpn.spec:110]: You want me to open a new PR to change this one line?

File: eduvpn/settings.py (hunk 2 above)
Comment (alvra) [reply eduvpn/settings.py:32]: `CLIENT_ID` is used for OAuth, but I'm not sure how that's used by servers.

File: eduvpn.spec (hunk 1 above)
Comment (gijzelaerr) [reply eduvpn.spec:110]: i just want it to be fixed. you can also push to this branch. it I snot one line, it is everywhere line and file in this PR where `letsconnect-client` has been changed in `org.letsconnect.client` where it should be `org.letsconnect-vpn.client`

File: eduvpn/settings.py (hunk 2 above)
Comment (gijzelaerr) [reply eduvpn/settings.py:32]: i guess it is unrelated.

File: eduvpn.spec (hunk 1 above)
Comment (gijzelaerr) [reply eduvpn.spec]: done myself.

**Pull request merged**